### PR TITLE
BIP-85: formatting and changelog updates, and add word cases for application 39'

### PR DESCRIPTION
**Pull request by akarve**
* Address lingering feedback from #1679 
* Complete words table for application 39' (I could bump to semver 1.4.0. for this but that seems excessive)
* Reorder sections to be more standard
* Convert discussion to standard footnote

### CODE REVIEW

File: bip-0085.mediawiki [hunk 1]
```diff
@@ -167,10 +196,18 @@ Words Table
...
+| 15'
+|-
 | 18 words
 | 192 bits
 | 18'
 |-
+| 21 words
+| 224 bits
+| 21'
+|-
```
Comment (jonatack) [inline bip-0085.mediawiki:210]: Ok, these reflect the table in BIP39 "Generating the mnemonic" section.

File: bip-0085.mediawiki [hunk 2]
```diff
@@ -95,7 +95,36 @@ OUTPUT
 
 * DRNG(80 bytes)=b78b1ee6b345eae6836c2d53d33c64cdaf9a696487be81b03e822dc84b3f1cd883d7559e53d175f243e4c349e822a957bbff9224bc5dde9492ef54e8a439f6bc8c7355b87a925a37ee405a7502991111
 
-==Reference Implementation==
+==Changelog==
```
Comment (jonatack) [inline bip-0085.mediawiki:98]: Maybe ~follow the following order, per that of other BIPs:

- Abstract
- Copyright
- Definitions / Motivation

.../...

- References
- Reference Implementation
- Changelog
- Footnotes
- Acknowledgements

**Review by jonatack — COMMENTED** [review]
Concept ACK

File: bip-0085.mediawiki (hunk 2 above)
Comment (jonatack) [reply bip-0085.mediawiki:98]: @akarve what do you think?

Comment (akarve) [reply bip-0085.mediawiki:98]: Sounds reasonable. Will put something together over the weekend.

**Review by jonatack — COMMENTED** [review]
ACK b398219e9cf23b66431debda13765f6557e98239

Only changes are presentation/formatting and adding two entries to the application 39' table.

File: bip-0085.mediawiki (hunk 2 above)
Comment (Karthik-0521) [reply bip-0085.mediawiki]: > @@ -95,7 +95,36 @@ OUTPUT * DRNG(80 bytes)=b78b1ee6b345eae6836c2d53d33c64cdaf9a696487be81b03e822dc84b3f1cd883d7559e53d175f243e4c349e822a957bbff9224bc5dde9492ef54e8a439f6bc8c7355b87a925a37ee405a7502991111 -==Reference Implementation== +==Changelog==

File: bip-0085.mediawiki (hunk 1 above)
Comment (Karthik-0521) [reply bip-0085.mediawiki:183]: > @@ -95,7 +95,36 @@ OUTPUT
 
 * DRNG(80 bytes)=b78b1ee6b345eae6836c2d53d33c64cdaf9a696487be81b03e822dc84b3f1cd883d7559e53d175f243e4c349e822a957bbff9224bc5dde9492ef54e8a439f6bc8c7355b87a925a37ee405a7502991111
 
-==Reference Implementation==
+==Changelog==

**Pull request merged**